pkg/services/login/authinfo.go

```diff
@@ -34,6 +34,8 @@ func GetAuthProviderLabel(authModule string) string {
 		return "AzureAD"
 	case "oauth_gitlab":
 		return "GitLab"
+	case "oauth_okta":
```

Comment:
this is different from previous, as it seems we didn't have okta part of the UI and the backend.

It was "OAuth" previously